src/node/package.json

```
@@ -29,6 +29,7 @@
   "author": "",
   "license": "AGPL-3.0-only",
   "dependencies": {
+    "https-proxy-agent": "^7.0.2",
```

We'd prefer to use the previous version for now, because that's what our desktop app is using (so we can avoid adding another version).

```suggestion
    "https-proxy-agent": "7.0.1",
```

---

Sure, done!